daemon/cluster/cluster.go

```diff
@@ -666,16 +666,22 @@ func (c *Cluster) Leave(force bool) error {
 		if c.locked {
 			c.locked = false
 			c.lastNodeConfig = nil
-			c.Unlock()
-		} else if c.err == ErrSwarmCertificatesExpired {
+		} else if c.err == errSwarmCertificatesExpired {
 			c.err = nil
 			c.Unlock()
 		} else {
 			c.Unlock()
-			return ErrNoSwarm
+			return errNoSwarm
 		}
 	} else {
 		if node.Manager() != nil && !force {
+			if c.locked {
+				// when swarm manager is locked and Leave action is without --force
+				// return error to not allow action Leave
+				c.Unlock()
+				return fmt.Errorf("Swarm is encrypted and locked. Leaving swarm without `--force` may cause data loss. Please unlock it first or use `--force` to ignore this message")
```

Comment:
Thanks, my friends. Now added a period. And skip that `Leaving swarm without --force may cause data loss.` I feel a bit confused as well.